### PR TITLE
Added RDS Custom for SQL Server example

### PR DESCRIPTION
**Pull request by AdamTylerLynch**
### Description

Added RDS Custom for SQL Server example.

### Relations

Relates #28765

### References

* https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/custom-cev-sqlserver.html

### CODE REVIEW

**Comment by AdamTylerLynch** [comment]
Not sure what is breaking. I'm heading on PTO for a month, can someone take a look at this?

File: website/docs/r/db_instance.html.markdown [hunk 1]
```diff
@@ -126,6 +126,45 @@ resource "aws_db_instance" "test-replica" {
...
+}
+
+# The RDS instance resource requires an ARN. Look up the ARN of the KMS key.
+data "aws_kms_key" "by_id" {
+  key_id = "example-ef278353ceba4a5a97de6784565b9f78" # KMS key
+}
+
+resource "aws_db_instance" "example" {
+  allocated_storage           = 500
+  auto_minor_version_upgrade  = false                               # Custom for Oracle not support minor version upgrades
```
Comment (ewbankkit) [inline website/docs/r/db_instance.html.markdown:147]:
```suggestion
  auto_minor_version_upgrade  = false                               # Custom for SQL Server does not support minor version upgrades
```

File: website/docs/r/db_instance.html.markdown [hunk 2]
```diff
@@ -83,8 +83,8 @@ data "aws_kms_key" "by_id" {
 
 resource "aws_db_instance" "default" {
   allocated_storage           = 50
-  auto_minor_version_upgrade  = false                         # Custom for Oracle not support minor version upgrades
-  custom_iam_instance_profile = "AWSRDSCustomInstanceProfile" # Instance profile is required for Custom for Oracle. See: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/custom-setup-orcl.html#custom-setup-orcl.iam-vpc
+  auto_minor_version_upgrade  = false                         # Custom for SQL Server not support minor version upgrades
+  custom_iam_instance_profile = "AWSRDSCustomInstanceProfile" # Instance profile is required for Custom for SQL Server. See: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/custom-setup-orcl.html#custom-setup-orcl.iam-vpc
```
Comment (ewbankkit) [inline website/docs/r/db_instance.html.markdown:87]:
```suggestion
  auto_minor_version_upgrade  = false                         # Custom for Oracle does not support minor version upgrades
  custom_iam_instance_profile = "AWSRDSCustomInstanceProfile" # Instance profile is required for Custom for Oracle. See: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/custom-setup-orcl.html#custom-setup-orcl.iam-vpc
```

File: website/docs/r/db_instance.html.markdown (hunk 1 above)
Comment (ewbankkit) [reply website/docs/r/db_instance.html.markdown:147]: DB engine to match section header.

**Comment by ewbankkit** [comment]
@AdamTylerLynch Thanks for the contribution :tada: :clap:.

**Pull request merged**